frontend/libs/studio-content-library/src/ContentLibrary/LibraryBody/pages/CodeList/CodeLists/CodeLists.tsx

```diff
@@ -20,16 +22,22 @@ type CodeListProps = {
 };
 
 function CodeList({ codeList, onUpdateCodeList }: CodeListProps) {
-  const { t } = useTranslation();
+  const editorTexts: CodeListEditorTexts = useOptionListEditorTexts();
+
+  const handleUpdateCodeList = (updatedCodeList: StudioComponentsCodeList) => {
+    onUpdateCodeList({ title: codeList.title, codeList: updatedCodeList });
+  };
```

Comment:
For å følge kodereglene våre, ville jeg flyttet denne databehandlingen ut i en egen funksjon. Da blir det enklere å skalere hvis vi senere endrer på `CodeListWithMetadata`-typen. Husk for øvrig returtype.
```suggestion
  const handleUpdateCodeList = (updatedCodeList: StudioComponentsCodeList): void => {
    const updatedCodeListWithMetadata = updateCodeList(currentCodeListWithMetadata, updatedCodeList);
    onUpdateCodeList(updatedCodeListWithMetadata);
  };
```